src/App.js

```diff
@@ -3,15 +3,15 @@ import { BrowserRouter as Router, Route, Routes } from 'react-router-dom';
 import './index.css';
 import Books from './pages/BookPage';
 import Categories from './pages/Categories';
-import Nav from './components/Nav';
+import Header from './components/Header';
 
 /* eslint-disable react/prop-types */
 /* eslint-disable  react/destructuring-assignment */
 function App() {
   return (
     <div>
       <Router>
-        <Nav />
+        <Header />
         <Routes>
           <Route path="/books" element={<Books />} />
```

Comment:
- [OPTIONAL] Please kindly make the Books component the display in the default path (`path="/"`) so that you will not have a blank page when your site loads. Something like so: 

```
 <Route path="/" element={<Books />} />
```